snorkel/classification/utils.py

```diff
@@ -72,6 +72,33 @@ def pad_batch(
     return padded_batch, mask_batch
 
 
+def get_active_mask(Y: torch.Tensor) -> torch.Tensor:
+    """Return a mask of the active samples given some label Tensor Y.
```

Comment:
Let's define 'active' at least once. i.e., it's active if at least one value in its row is not ABSTAIN.